### PR TITLE
Feature/shopping cart page default

### PR DESCRIPTION
**Pull request by barbigrade**
- Create a new page for the shopping cart
- Upload background image(s) and shopping basket icon
- Create a primary button component
- Create an empty shopping cart component and add it to its corresponding page
- Add basic styling

### CODE REVIEW

File: components/ShoppingCartCard.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import Image from 'next/image';
+
+export default function ShoppingCartCard({ heading, description, CTA }) {
+  return (
+    <ShoppingCartWrapper>
+      <Heading>{heading}</Heading>
+      <Image
+        alt=" "
```
Comment (ThomasKirst) [inline components/ShoppingCartCard.js:9]:
```suggestion
        alt="Shopping cart icon"
```

File: pages/shoppingcart.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import Image from 'next/image';
+import styled from 'styled-components';
+import PrimaryButton from '../components/PrimaryButton';
+import ShoppingCartCard from '../components/ShoppingCartCard';
+
+export default function ShoppingCartPage() {
+  return (
+    <>
```
Comment (ThomasKirst) [inline pages/shoppingcart.js:8]: The outer fragment is not needed here.

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Very nice and concise, your components :)
I just left two tiny remarks / suggestions.
Keep it up 🥐

**Comment by barbigrade** [comment]
Thank you for checking @ThomasKirst ! I just made the changes you suggested, hope it's fine now 🥨

**Pull request merged**